src/engine/BMGame.php

```diff
@@ -4084,7 +3851,7 @@ protected function setBMPlayerProps($property, array $value, $subclass = '') {
     /**
      * Set all players' waitingOnAction statuses to FALSE
      */
-    protected function setAllToNotWaiting() {
+    public function setAllToNotWaiting() {
```

Comment:
I don't know that I agree that this function shouldn't be public, since it would be possible to replace the function call with the function contents, and that wouldn't break any of PHP's (broken) encapsulation. However, I'll take a closer look at it.